src/XMLexport.cpp

```diff
@@ -555,8 +555,10 @@ bool XMLexport::writeHost( Host * pHost )
             if( ! (*it) || (*it)->mModuleMember) {
```

Comment:
Can include the temp key check here instead as:

    if( ! (*it) || (*it)->isTempKey() || (*it)->mModuleMember) {